config/metricConfigs/skillAssessmentMetrics.ts

```diff
@@ -0,0 +1,17 @@
+import { MetricsConfig } from '../../src/aggregators/util';
+
+const allowedMetrics: MetricsConfig = [
+  {
+    name: 'sa_check_item',
+    help: 'Time it takes to check a skill assessment item',
+    type: 'histogram',
+    labels: [
+      {
+        name: 'appName',
+        allowedValues: ['skill-assessment'],
+      },
+    ],
+  },
```

Comment:
you also need to include the `protocol` attribute like in this file https://github.com/datacamp/prometheus-user-metrics/blob/master/config/metricConfigs/learnMetrics.ts
You can set it to `statsd`

Reply:
@ludov04 Right! Fixed it.